cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -52,6 +52,7 @@ var (
 	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
 	argKubecfgFile         = flag.String("kubecfg_file", "", "Location of kubecfg file for access to kubernetes master service; --kube_master_url overrides the URL part of this; if neither this nor --kube_master_url are provided, defaults to service account tokens")
 	argKubeMasterURL       = flag.String("kube_master_url", "", "URL to reach kubernetes master. Env variables in this flag will be expanded.")
+	argEnablePodDNS        = flag.Bool("enable-pod-dns", true, "If set to true, pods will get DNS within the cluster.")
```

Comment:
> We don't "have" to, I just did in case someone wanted to turn off pods to mimic the current behavior. Should I remove?

less is more!

we'll have to follow this up with a PR to bump versions and a container push.